travis/travis_install_nightly

```diff
@@ -68,7 +68,16 @@ fi
 # Workaround to force using system site packages (see https://github.com/Shippable/support/issues/241#issuecomment-57947925)
 rm -f $VIRTUAL_ENV/lib/python2.7/no-global-site-packages.txt
 pip install --upgrade pip
-pip install --user -q --no-binary pycparser -r $(dirname ${BASH_SOURCE[0]})/requirements.txt
```

Comment:
@moylop260 This PR has broken l10n-spain: https://travis-ci.org/OCA/l10n-spain/jobs/250811101 and I think the problem is in this line.